protocol/metrics/consumer_relayserver_client.go

```diff
@@ -111,19 +113,21 @@ func (cuc *ConsumerRelayServerClient) SetRelayMetrics(relayMetric *RelayMetrics)
 	cuc.appendQueue(request)
 }
 
-func (cuc *ConsumerRelayServerClient) aggregateAndSendRelayData(sendQueue []UpdateMetricsRequest, sendID int, cucEndpointAddress string) (*http.Response, error) {
+func (cuc *ConsumerRelayServerClient) aggregateAndSendRelayData(sendQueue []UpdateMetricsRequest, sendCnt int, cucEndpointAddress string) (*http.Response, error) {
+	utils.LavaFormatDebug("[CUC] Sending data to server - start ", utils.LogAttr("Size of metrics to send", len(sendQueue)))
```

Comment:
please dont print every time you aggregate, just when you send

Reply:
i was debugging it with nenad last time and starting to send was also valuable infromation - then you see if the problem that the sending takes 2 much time
what are my options to turn them on/off for debugging ?